### PR TITLE
Added tenant details page

### PR DESCRIPTION
**Pull request by rixong**

### CODE REVIEW

File: tenant-file/portal-app/src/component/tenant-list.tsx [hunk 1]
```diff
@@ -131,7 +144,7 @@ const TenantList: React.FC = () => {
           prepareRow(row);
           return (
             <>
-              <tr {...row.getRowProps()}>
+              <tr {...row.getRowProps()} onClick={onHandleRowClick} data-user-id={mockUserIdNumber}>
```
Comment (ryanrightmer) [inline tenant-file/portal-app/src/component/tenant-list.tsx:147]: Can you use styled-components to add a `cursor: pointer;` to the row?

**Review by ryanrightmer — CHANGES_REQUESTED** [review]
Looks pretty good, let's just get those changes in and I can merge this

File: tenant-file/portal-app/src/component/dashboard.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import TenantList from './tenant-list';
 import { Container } from 'reactstrap';
+// import PrivateRoute from './private-route';
```
[the remark this answers is not in the document]
Comment (rixong) [reply tenant-file/portal-app/src/component/dashboard.tsx:4]: done.

File: tenant-file/portal-app/src/component/tenant-details.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {RouteComponentProps} from 'react-router-dom';
+
+type TParams = {
+  id: string;
+}
+
+const TenantDetails = ({ match }: RouteComponentProps<TParams>) => {
```
[the remark this answers is not in the document]
Comment (rixong) [reply tenant-file/portal-app/src/component/tenant-details.tsx:8]: done.

File: tenant-file/portal-app/src/component/tenant-list.tsx (hunk 1 above)
Comment (rixong) [reply tenant-file/portal-app/src/component/tenant-list.tsx:147]: I had added hover effect (line 132) to table. Do you want to leave the hover shading as well as pointer?
I Just added cursor at 147: 
<tr style={{cursor:'pointer'}} {...row.getRowProps()} onClick={onHandleRowClick} data-user-id={mockUserIdNumber}>

**Review by ryanrightmer — APPROVED** [review]

**Pull request merged**